master/getting-started/kubernetes/tutorials/stars-policy/index.md

```diff
@@ -87,3 +87,5 @@ kubectl create -f policies/frontend-policy.yaml
 
 The client can now access the frontend, but not the backend.  Neither the frontend nor the backend
 can initiate connections to the client.  The frontend can still access the backend.
+
+For advanced network policy demonstration, check out the [advanced network policy demo]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/tutorials/advanced-policy).
```

Comment:
I'd make this more specific - "To use Calico to enforce egress policy on Kubernetes pods, see the advanced policy demo"